### PR TITLE
Add CTAN integration

### PR DESCRIPTION
**Pull request by raphink**
This adds basic support (version and license) for CTAN (TeX/LaTeX) package badges.

### CODE REVIEW

**Comment by raphink** [comment]
@espadrine are you interested in this PR?

**Comment by espadrine** [comment]
@raphink Definitely! I'll try to review all the PRs in the coming days.

File: server.js [hunk 1]
```diff
@@ -2929,6 +2929,45 @@ cache(function(data, match, sendBadge, request) {
...
+  var badgeData = getBadgeData('ctan', data);
+  request(url, function (err, res, buffer) {
+    if (err != null) {
+      badgeData.text[1] = 'inaccessible';
+      sendBadge(badgeData, format);
+    }
+    try {
+      var data = JSON.parse(buffer);
+
+      if (info == 'v') {
```
Comment (espadrine) [inline server.js]: Can you switch to triple equals please? `info === 'v'`

Comment (raphink) [inline server.js]: Why is that?

**Comment by raphink** [comment]
(I've now used `===` in all my PRs)

**Comment by espadrine** [comment]
Splendid work! Thanks a lot!

I'll get around to the other PR tomorrow. Long day.

**Comment by raphink** [comment]
Great! I'll be looking forward to your reviewing the other PRs!

**Pull request merged**